mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -167,6 +167,17 @@ in Terraform state, a `terraform destroy` or `terraform apply` that would delete
 * `table_constraints` - (Optional) Defines the primary key and foreign keys. 
     Structure is [documented below](#nested_table_constraints).
 
+* `resource_tags` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
+    This field is in beta. The tags attached to this table. Tag keys are
+    globally unique. Tag key is expected to be in the namespaced format, for
+    example "123456789012/environment" where 123456789012 is the ID of the
+    parent organization or project resource for this tag key. Tag value is
+    expected to be the short name, for example "Production".
+
+* `allow_resource_tags_on_deletion` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
+    This field is in beta. Whether or not to allow table deletion when there are
```

Comment:
nit(optional): Maybe a language like "Setting this to *true* will allow table deletion when there are still resource tags attached." is a tad clearer.

Reply:
I'd suggest this as well. I'm going to merge this now anyways, but it can be done in a followup if you'd like!